content/solana-nft.md

```diff
@@ -62,14 +62,14 @@ Before creating a collection you must prepare the assets for your collection. Th
 
 The metadata should include the following attributes:
 
-- `"name"` the name of the NFT that will display on-chain
-- `"symbol"` the optional symbol of the NFT that will display on-chain
-- `"description"` the description of NFT
-- `"seller_fee_basis_points"` the fee collected on sale of the NFT split between the creators
-- `"image"` the file name of the corresponding image that the NFT will display
-- `"attributes"` the attributes of the NFT
-- `"properties"` the creators that share of seller fee basis points. If there multiple creators, the total “share” must add to 100
-- `"collection"` the “name” and “family” of the NFT collection
+- `name` the name of the NFT that will display on-chain
+- `symbol` the optional symbol of the NFT that will display on-chain
+- `description` the description of the NFT
+- `seller_fee_basis_points` the fee collected upon sale of the NFT which is split between the creators
+- `image` the file name of the corresponding image that the NFT will display
+- `attributes` the attributes of the NFT
+- `properties` the creators that share of seller fee basis points. If there are multiple creators, the total “share” must add to 100
```

Comment:
@ZYJLiu  the "properties" explanation is awkward: "... the _**creators that share of**_ seller fee basis points."

Should it be
1. the creators that share seller fee basis points
or
2. the creator’s share of seller fee basis points

?